### PR TITLE
Also publish sources of android AAR

### PR DESCRIPTION
**Pull request by petter-fidesmo**

### CODE REVIEW

**Comment by petter-fidesmo** [comment]
@slomo Ready for review!

File: android/build.sbt [hunk 1]
```diff
@@ -15,5 +15,3 @@ autoScalaLibrary := false
 javacOptions in (Compile, compile) ++= Seq("-source", "1.6", "-target", "1.6", "-Xlint:unchecked", "-Xlint:deprecation")
 
 publishArtifact in (Compile, packageDoc) := false
```
Comment (slomo) [inline android/build.sbt]: Why no doc package?

Comment (petter-fidesmo) [inline android/build.sbt]: That breaks the build. Do you have any idea of how we could get that to work

**Comment by slomo** [comment]
:+1:

**Pull request merged**